azure-container-apps-maven-plugin/README.md

```diff
@@ -0,0 +1,23 @@
+# Maven Plugin for Azure Container Apps
```

Comment:
Might be better to follow the webapp / function's `README.md` structure, given that their plugin is more frequently used and updated.

We can keep the `Azure Container Apps` description section, and add other missed sections.

Reply:
Function's plugin has the same structure and webapp's has more info. We can fill these part after the detailed doc ready if needed.